Packs/PowershellRemoting/Integrations/PowerShellRemoting/PowerShellRemoting.yml

```diff
@@ -3,16 +3,17 @@ commonfields:
   id: PowerShell Remoting
   version: -1
 configuration:
-- additionalinfo: Fully Qualified Domain Name suffix to be added to the hostname.
+- additionalinfo: Fully Qualified Domain Name suffix to be added to the hostname. For example mydomain.local
   display: Domain
   name: domain
   required: false
   type: 0
 - display: DNS
+  additionalinfo: The IP address of the DNS server the integration will use to resolve your hosts.
```

Comment:
```suggestion
  additionalinfo: Optional IP address of a DNS server the integration will use to resolve your hosts. Use this setting when the host linux machine uses a different DNS from the Windows machines you are targeting.
```

Reply:
@DeanArbel Do you recall if this will actually work for hostnames if we dont provide an DNS server? will the docker use the server to resolve?